test/unit/gateways/paypal_express_test.rb

```diff
@@ -39,13 +35,11 @@ def teardown
   def test_live_redirect_url
     Base.mode = :production
     assert_equal LIVE_REDIRECT_URL, @gateway.redirect_url_for('1234567890')
-    assert_equal LIVE_REDIRECT_URL_MOBILE, @gateway.redirect_url_for('1234567890', :mobile => true)
```

Comment:
This goes away because with express in context, there is no mobile-specific URL, right?